### PR TITLE
Make test suite pass

### PR DESCRIPTION
**Pull request by doudou**

### CODE REVIEW

**Comment by g-arjones** [comment]
Great! :+1: x 1000

File: lib/autoproj/reporter.rb [hunk 1]
```diff
@@ -66,21 +66,24 @@ def reset_timer
...
                     "#{val} #{val > 1 ? name + 's' : name}"
                 end
             end.compact.reverse.join(' ')
         end
 
         def success
             elapsed_string = elapsed_time ? " (took #{elapsed_time})" : ''
-            Autoproj.message("Command finished successfully at #{Time.now}#{elapsed_string}", :bold, :green)
+            Autoproj.message("Command finished successfully at "\
+                             "{Time.now}#{elapsed_string}", :bold, :green)
```
Comment (g-arjones) [inline lib/autoproj/reporter.rb]:
```suggestion
                             "#{Time.now}#{elapsed_string}", :bold, :green)
```

**Comment by 2maz** [comment]
Couple of tests fail on my system (bionic), related to having ruby-bundler installed as osdeps.
Maybe add ruby-bundler as default osdeps to CI to expose this and cover for this?

```
gem install bundler returned successfully, but still cannot find bundler in /home/rock-user/.autoproj/gems/ruby/2.5.0
``` 
```
$>gem2.5 which -a bundler/setup
/usr/lib/ruby/vendor_ruby/bundler/setup.rb
```

```
$>RUBYLIB=/home/rock-user/.autoproj/gems/ruby/2.5.0/gems/bundler-2.0.1/lib gem2.5 which -a bundler/setup
/home/rock-user/.autoproj/gems/ruby/2.5.0/gems/bundler-2.0.1/lib/bundler/setup.rb
/usr/lib/ruby/vendor_ruby/bundler/setup.rb
```

**Comment by doudou** [comment]
> Maybe add ruby-bundler as default osdeps to CI to expose this and cover for this?

We should definitely have one configuration with ruby-bundler installed and one without. Can you take care of that ?

**Comment by doudou** [comment]
OK. I'm merging this. @2maz can you take care of setting up CI for the ruby-bundler issue ?

**Comment by doudou** [comment]
@2maz is this an issue on the current release ? If it does, I'm keen on releasing the current version. Otherwise, I'll consider it a blocker.

**Comment by 2maz** [comment]
Looks to me like a problem of the test setup. Standard bootstrap works just fine. Ok to release from my side.

Btw: there might be some debug statement left (OPT[]) or is this intentional?
https://github.com/rock-core/autoproj/blob/08ba598ae6173994e903539f97655e379bee8c70/bin/autoproj_bootstrap#L750

**Comment by doudou** [comment]
> Btw: there might be some debug statement left (OPT[]) or is this intentional?

Definitely not. Fixed

**Pull request merged**